### DISCUSSION TITLE
New format for Webhook URLs Used by MS Teams

### DISCUSSION THREAD

**Discussion by kejdajar** [containrrr/watchtower#848] [Q&A] [answered]
Hello,
the currently supported format for _msteams_ notification hook according to the [docs](https://containrrr.dev/watchtower/notifications/#microsoft_teams) is:
* ```https://outlook.office.com/webhook/xxxxxxxx@xxxxxxx/IncomingWebhook/yyyyyyyy/zzzzzzzzzz```

[Here](https://office365itpros.com/2021/02/03/new-format-webhook-url/) you can find more information about the new webhook format.
Right now, MsTeams already generate the webhook using the new syntax:
- ``` https://YOUR_COMPANY.webhook.office.com/webhookb2/7aa49aa6-7840-443d-806c-08ebe8f59966@c662313f-14fc-43a2-9a7a-d2e27f4f3478/IncomingWebhook/8592f62b50cf41b9b93ba0c0a00a0b88/eff4cd58-1bb8-4899-94de-795f656b4a18```

Currently, the older version supported by Watchtower is still accepted by the MsTeams service, but I am afraid the backward compatibility will stop working by April 11, 2021.

Are there any plans to support the new format for incoming webhook URLs used by MsTeams?

**Comment by piksel** [comment, marked as answer]
Yes, it's in [shoutrrr v0.4](https://github.com/containrrr/watchtower/pull/810).
